.changelog/unreleased/dependencies/2116-cosmossdk-io-log-1-4-0.md

```diff
@@ -0,0 +1 @@
+* `cosmossdk.io/log` bumped to v1.4.0 (from v1.3.1) [PR 2116](https://github.com/provenance-io/provenance/pull/2116).
```

Comment:
WooHoo!

When this PR was first created, there was a second indirect library bump. But that library was also bumped in another PR that was merged first. When I updated this PR with `main`, it reran the changelog action which generated new content for this file, without the 2nd entry that is no longer being bumped in this PR.

I just thought it was cool to see that in action to make sure it worked as expected.